infra/notifications/README.md

```diff
@@ -87,5 +97,13 @@ For more (outdated) information, see [the original pull request](https://github.
 
 For local testing, these options are usual for testing against `rinkeby` network.
 
-    npm run start:listener:development -- --identity --growth --marketplace --network=rinkeby --continue-file=../continue --webhook=http://localhost:3456/events --verbose --concurrency=4
+````
+export DATABASE_URL=postgresql://origin@localhost/origin
 
+npm run start:listener:development -- --identity --growth --marketplace \
+  --network=rinkeby \
+  --continue-file=../continue \
+  --notifications-webhook=http://localhost:3456/events \
+  --verbose \
+  --concurrency=4
```

Comment:
I would discourage the use of concurrency, except when running backfills.
I think weird things could happen when concurrency is on - it makes listener process events without any ordering guarantee. This hasn't been tested well.

Reply:
Sounds good, no need to complicate.  Removed.